### ISSUE TITLE
test-netlink fails on beaglebone_black

### ISSUE THREAD

**Issue by Sangeetha-E** [systemd/systemd#27969]
Hi,

The test-netlink fails on beaglebone_black.

Here is the error log.

```
FAIL: test-netlink
/* test_rtnl_set_link_name */
/* test_genl */
/* test_message_strv */
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
sd-netlink: message parse - overwriting repeated attribute
/* test_message_array */
/* test_sd_netlink_message_get_errno */
/* test_dump_addresses */
got IPv4 address on ifindex 1
got IPv4 address on ifindex 3
got IPv6 address on ifindex 1
got IPv6 address on ifindex 3
/* test_sd_netlink_add_match */
/* test_message_container */
/* test_pipe */
1 left in pipe. got reply: Success
0 left in pipe. got reply: Success
/* test_async_destroy_callback */
link_handler2: got link info about lo
test_async_object_destroy: n_ref=2
test_async_object_destroy: n_ref=2
test_async_object_destroy: n_ref=2
/* test_netlink_call_async */
link_handler: got link info about lo
/* test_netlink_event_loop */
link_handler: got link info about lo
/* test_message_route */
/* test_message_address */
/* test_message_getlink */
Assertion 'sd_netlink_message_read_u32(reply, IFLA_NUM_RX_QUEUES, &u32_data) >= 0' failed at src/libsystemd/sd-netlink/test-netlink.c:79, function test_message_getlink(). Aborting.
```

Kindly give me suggestions to fix this failure.

systemd: 253
Linux kernel: 5.10.100

Thank you.

**Comment by poettering** [comment]
what kind of environment did you invoke this in? on the host? in a container manager? some build system?

which distro is this?

**Comment by Sangeetha-E** [comment]
I ran this test on target board(beaglebone_black).

**Comment by poettering** [comment]
i don't know what "target board" means. But again, is this run on the host or in some container? on which distro?

**Comment by Sangeetha-E** [comment]
I cross compiled the systemd and ran on other container not host.

The beaglebone_balck board is arm architecture.

**Comment by poettering** [comment]
i am sorry, but this is not a support forum, your issue is not actionable to us, if you don't answer the questions

**Comment by Sangeetha-E** [comment]
Sorry for this inconvenience. I used build system like buildroot but it is not distro specific.

**Comment by yuwata** [comment]
I guess the kernel is built with CONFIG_RPS disabled.
https://github.com/torvalds/linux/blob/64569520920a3ca5d456ddd9f4f95fc6ea9b8b45/net/core/rtnetlink.c#L1821

**Comment by yuwata** [comment]
Fix is waiting in #27993.

**Comment by Sangeetha-E** [comment]
The Issue resolved. Thank you!
